packages/@okta/vuepress-site/docs/guides/event-hook-ngrok/main/index.md

```diff
@@ -26,6 +28,17 @@ This guide provides a working example of an Okta Event Hook. It uses the ngrok u
 
 ---
 
+## Overview
+
+Event Hooks are outbound calls from Okta that can notify your own software systems of events occurring in your Okta org. The ngrok utility exposes local applications to the internet and receives and responds to Event Hook calls. This setup enables the development and testing of Event Hooks locally, rather than using an external test or production environment. The ngrok utility also provides replay functionality that assists in testing and developing your external service code.
```

Comment:
```suggestion
Event Hooks are outbound calls from Okta that can notify your software systems of events that occurred in your Okta org. The ngrok utility exposes local applications to the internet and receives and responds to Event Hook calls. This setup enables the development and testing of Event Hooks locally, rather than using an external test or production environment. The ngrok utility also provides a replay function to assist you in testing and developing your external service code.
```